code/API_definitions/device-roaming-status-subscriptions.yaml

```diff
@@ -1689,7 +1689,7 @@ components:
         sink: "https://endpoint.example.com/sink"
         sinkCredential: {
           "credentialType": "ACCESSTOKEN",
-          "accessToken": "JhbGciOiJSUzI1NiIsImtpZCI6Imk2bEdrM0ZaenhSY1ViMkMzbkVRN3N5SEpsW",
+          "accessToken": "xxx",
```

Comment:
This will be fixed by #17, but probably it does not matter to fix it here as well